trunner/device.py

```diff
@@ -35,6 +36,15 @@ def is_github_actions():
     return os.getenv('GITHUB_ACTIONS', False)
 
 
+def save_in_logfile(threshold_exceeded=False, wait_time=0):
+    if not os.path.exists(LOGFILE_PATH):
+        os.system(f'echo "Does wait time exceed 5s?" > {LOGFILE_PATH}')
```

Comment:
no, please open file in python and write directly instead of unnecessary process forking here.
(**this needs to be fixed before the merge**)

I would (always) store raw restart times in file.csv to be able to compute max/min/mean/avg and quickly plot graph from it, the file doesn't need to be human-readable.